### PR TITLE
i18n(zh-cn): add guides/migrate-to-astro/from-sveltekit.mdx

### PR DESCRIPTION
**Pull request by zhuchentong**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- add guides/migrate-to-astro/from-sveltekit.mdx

### CODE REVIEW

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## SvelteKit 和 Astro 的主要相似之处
+
+Next.js 和 Astro 有一些相似之处，可以帮助你迁移你的项目：
+
+- SvelteKit 和 Astro 都是现代 JavaScript 静态站点生成器和服务器端渲染框架。
+
+- SvelteKit 和 Astro 都存在一个 [`src/` 文件目录在您的项目文件](/zh-cn/core-concepts/project-structure/#src) 以及一个 [基于文件路由的特定文件目录](/zh-cn/core-concepts/astro-pages/)。对于您创建和管理页面应该会感觉很熟悉。
+
+- Astro 有[一个使用Svelte组件的官方集成](/zh-cn/guides/integrations-guide/svelte/)，并支持[安装NPM包](/zh-cn/guides/imports/#npm-packages)，包括一些用于 Svelte 的包。您将能够编写 Svelte UI 组件，并且可能做到保留部分或全部现有的组件和依赖项。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:23]:
```suggestion
- Astro 有[一个使用Svelte组件的官方集成](/zh-cn/guides/integrations-guide/svelte/)，并支持[安装NPM包](/zh-cn/guides/imports/#npm-包)，包括一些用于 Svelte 的包。您将能够编写 Svelte UI 组件，并且可能做到保留部分或全部现有的组件和依赖项。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+- [以内容为中心](/zh-cn/concepts/why-astro/)：Astro 的设计目标是能够在以内容为中心的网站上有更优异的表现。现有的 SvelteKit 应用程序可能是为客户端的高交互性而构建的，并且可能包含诸如仪表板之类的页面，或者使用 SvelteKit 的 [表单操作](https://kit.svelte.dev/docs/form-actions) 等功能，这些功能在 Astro 中实现起来更困难一些。
+
+- [Markdown 准备](/zh-cn/guides/markdown-content/)：Astro 包含内置的 Markdown 支持，并且包含用于每个文件的页面模板的一个 [特殊的前置 YAML `layout` 属性](/zh-cn/core-concepts/layouts/#markdownmdx-layouts)。如果您正在将 SvelteKit 的基于 Markdown 的博客转换为 Astro，您将不需要安装单独的 Markdown 集成，并且您不需要通过配置文件设置布局。您可以将现有的 Markdown 文件带入，但是您可能需要重新整理下这些文件，因为 Astro 的基于文件的路由不需要为每个页面路由创建一个文件夹。
+
+## 从 SvelteKit 切换到 Astro
+
+要将 SvelteKit 博客转换为 Astro，请从我们的博客主题起始模板开始，或者在我们的[主题展示](https://astro.build/themes/)中探索更多社区博客主题。
+
+您可以将 `--template` 参数传递给 `create astro` 命令，以便使用其中一个模板开始一个新的 Astro 项目。或者，您可以[从 GitHub 上的任何现有的 Astro 的存储库中开始一个新项目](/zh-cn/install/auto/#starter-templates)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:43]:
```suggestion
您可以将 `--template` 参数传递给 `create astro` 命令，以便使用其中一个模板开始一个新的 Astro 项目。或者，您可以[从 GitHub 上的任何现有的 Astro 的存储库中开始一个新项目](/zh-cn/install/auto/#入门模板)。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[SvelteKit](https://kit.svelte.dev) 是一个基于 Svelte 构建Web应用程序的框架。
+
+## SvelteKit 和 Astro 的主要相似之处
+
+Next.js 和 Astro 有一些相似之处，可以帮助你迁移你的项目：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:17]: Next?

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+- SvelteKit 和 Astro 都是现代 JavaScript 静态站点生成器和服务器端渲染框架。
+
+- SvelteKit 和 Astro 都存在一个 [`src/` 文件目录在您的项目文件](/zh-cn/core-concepts/project-structure/#src) 以及一个 [基于文件路由的特定文件目录](/zh-cn/core-concepts/astro-pages/)。对于您创建和管理页面应该会感觉很熟悉。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:21]:
```suggestion
- SvelteKit 和 Astro 都存在一个 [`src/` 文件目录在你的项目文件](/zh-cn/core-concepts/project-structure/#src) 以及一个 [基于文件路由的特定文件目录](/zh-cn/core-concepts/astro-pages/)。对于你创建和管理页面应该会感觉很熟悉。
```
"您" -> "你"

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+- Astro 有[一个使用Svelte组件的官方集成](/zh-cn/guides/integrations-guide/svelte/)，并支持[安装NPM包](/zh-cn/guides/imports/#npm-包)，包括一些用于 Svelte 的包。您将能够编写 Svelte UI 组件，并且可能做到保留部分或全部现有的组件和依赖项。
+
+- Astro 和 SvelteKit 都允许您使用 [无头 CMS、API 或 Markdown 文件进行数据获取](/zh-cn/guides/data-fetching/)。您可以继续使用您喜欢的 [创作内容](/zh-cn/guides/content/) 系统，并且能够保留您现有的内容。
+
+## SvelteKit 和 Astro 的主要不同之处
+
+当您将 SvelteKit 网站重构为 Astro 时，您会注意到一些重要的不同之处：
+
+- [MPA vs alternatives](/zh-cn/concepts/mpa-vs-spa/)：Astro 网站是多页面应用程序，而 SvelteKit 默认为支持服务器端渲染的 SPA（单页面应用程序），但也可以创建 MPA、传统 SPA，或者您可以在应用程序中混合使用这些技术。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:31]:
```suggestion
- [MPA vs 其它方案](/zh-cn/concepts/mpa-vs-spa/)：Astro 网站是多页面应用程序，而 SvelteKit 默认为支持服务器端渲染的 SPA（单页面应用程序），但也可以创建 MPA、传统 SPA，或者您可以在应用程序中混合使用这些技术。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+- [组件](/zh-cn/core-concepts/astro-components/)：SvelteKit 使用 [Svelte](https://svelte.dev)。Astro 页面使用 [`.astro` 组件](/zh-cn/core-concepts/astro-components/) 构建，但也可以支持 [React、Preact、Vue.js、Svelte、SolidJS、AlpineJS、Lit](/zh-cn/core-concepts/framework-components/) 和原始 HTML 模板。
+
+- [以内容为中心](/zh-cn/concepts/why-astro/)：Astro 的设计目标是能够在以内容为中心的网站上有更优异的表现。现有的 SvelteKit 应用程序可能是为客户端的高交互性而构建的，并且可能包含诸如仪表板之类的页面，或者使用 SvelteKit 的 [表单操作](https://kit.svelte.dev/docs/form-actions) 等功能，这些功能在 Astro 中实现起来更困难一些。
+
+- [Markdown 准备](/zh-cn/guides/markdown-content/)：Astro 包含内置的 Markdown 支持，并且包含用于每个文件的页面模板的一个 [特殊的前置 YAML `layout` 属性](/zh-cn/core-concepts/layouts/#markdownmdx-布局)。如果您正在将 SvelteKit 的基于 Markdown 的博客转换为 Astro，您将不需要安装单独的 Markdown 集成，并且您不需要通过配置文件设置布局。您可以将现有的 Markdown 文件带入，但是您可能需要重新整理下这些文件，因为 Astro 的基于文件的路由不需要为每个页面路由创建一个文件夹。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:37]:
```suggestion
- [Markdown 支持就绪](/zh-cn/guides/markdown-content/)：Astro 包含内置的 Markdown 支持，并且包含用于每个文件的页面模板的一个 [特殊的前置 YAML `layout` 属性](/zh-cn/core-concepts/layouts/#markdownmdx-布局)。如果您正在将 SvelteKit 的基于 Markdown 的博客转换为 Astro，您将不需要安装单独的 Markdown 集成，并且您不需要通过配置文件设置布局。您可以将现有的 Markdown 文件带入，但是您可能需要重新整理下这些文件，因为 Astro 的基于文件的路由不需要为每个页面路由创建一个文件夹。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]
The word "您" needs to be changed to "你" throughout the text.

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx (hunk 4 above)
Comment (zhuchentong) [reply src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx]: Okay, I'll fix that

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+## SvelteKit 和 Astro 的主要相似之处
+
+SvelteKit 和 Astro 有一些相似之处，可以帮助你迁移你的项目：
+
+- SvelteKit 和 Astro 都是现代 JavaScript 静态站点生成器和服务器端渲染框架。
+
+- SvelteKit 和 Astro 都存在一个 [`src/` 文件目录在你的项目文件](/zh-cn/core-concepts/project-structure/#src) 以及一个 [基于文件路由的特定文件目录](/zh-cn/core-concepts/astro-pages/)。为你的网站创建和管理页面应该会感觉很熟悉。
+
+- Astro 有[一个使用Svelte组件的官方集成](/zh-cn/guides/integrations-guide/svelte/)，并支持[安装NPM包](/zh-cn/guides/imports/#npm-包)，包括一些用于 Svelte 的包。你将能够编写 Svelte UI 组件，并且可能做到保留部分或全部现有的组件和依赖项。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:21]:
```suggestion
- Astro 有[一个使用 Svelte 组件的官方集成](/zh-cn/guides/integrations-guide/svelte/)，并支持[安装NPM包](/zh-cn/guides/imports/#npm-包)，包括一些用于 Svelte 的包。你将能够编写 Svelte UI 组件，并且可能做到保留部分或全部现有的组件和依赖项。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+- Astro 和 SvelteKit 都允许你使用 [无头 CMS、API 或 Markdown 文件进行数据获取](/zh-cn/guides/data-fetching/)。你可以继续使用你喜欢的 [创作内容](/zh-cn/guides/content/) 系统，并且能够保留你现有的内容。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:23]:
```suggestion
- Astro 和 SvelteKit 都允许你使用 [无头（headless） CMS、API 或 Markdown 文件进行数据获取](/zh-cn/guides/data-fetching/)。你可以继续使用你喜欢的 [创作内容](/zh-cn/guides/content/) 系统，并且能够保留你现有的内容。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+title: 从 SvelteKit 迁移
+description: 将现有的 SvelteKit 项目迁移到 Astro 的提示
+type: migration
+stub: true
+framework: SvelteKit
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[SvelteKit](https://kit.svelte.dev) 是一个基于 Svelte 构建Web应用程序的框架。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-sveltekit.mdx:11]:
```suggestion
[SvelteKit](https://kit.svelte.dev) 是一个基于 Svelte 构建 Web 应用程序的框架。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]
A little change.

**Review by liruifengv — APPROVED** [review]
LGTM. Thanks @zhuchentong

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for this! 💜

**Pull request merged**